ui/perfherder/compare/CompareSelectorView.jsx

```diff
@@ -130,15 +176,29 @@ export default class CompareSelectorView extends React.Component {
             )}
             <Row className="justify-content-center">
               <Col sm="8" className="text-right px-1">
-                <Button
-                  color="info"
-                  className="mt-2 mx-auto"
-                  onClick={
-                    newRevision !== '' && disableButton ? null : this.submitData
-                  }
-                >
-                  Compare
-                </Button>
+                <ButtonToolbar className="float-right">
+                  <UncontrolledDropdown className="mt-2 mr-2 text-nowrap">
```

Comment:
This button looks a bit odd next to the 'compare' button. Can you move it to the same row as the Projects button and give it the `outline` prop so it'll look the same.

Reply:
> This button looks a bit odd next to the 'compare' button. Can you move it to the same row as the Projects button and give it the `outline` prop so it'll look the same.

It's doable. But wouldn't this add a bit too much logic for the SelectorCard component?
It makes it less reusable, as it requires extra logic & state to show the dropdown on the "New" SelectorCard & hide it on the "Base" one.

@sarah-clements maybe merging the framework dropdown with the Compare button using something like `InputGroupAddon` would solve some of this ambiguity?

Reply:
I see what you mean about reusability. One way to make it more flexible is to refactor the dropdown menu in the SelectorCard so that it'll loop over an array of objects that supplies all of the properties needed for n dropdown menu's. I've done something similar throughout Perfherder. You could probably just use the `createDropdowns` function: https://github.com/mozilla/treeherder/blob/master/ui/perfherder/FilterControls.jsx#L17

Which was used here: https://github.com/mozilla/treeherder/blob/master/ui/perfherder/graphs/TestDataModal.jsx#L261

What do you think?

Not sure I follow what you mean about the `InputGroupAddon - could you post a screen shot?

Reply:
> Not sure I follow what you mean about the `InputGroupAddon` - could you post a screen shot?

@sarah-clements sure. This is how the controls would look if merged together.
The default will remain `talos`, but the user can select something else when needed.
I think it's an intuitive control.
![merged-controls](https://user-images.githubusercontent.com/12194756/62940993-92cf6c00-bddd-11e9-8ce3-7dd653e9ebae.png).

Reply:
That's an interesting idea. It still seems a bit strange to me to have projects in the card and frameworks as part of the compare button. I suppose you can try it but perhaps switching it so the framework dropdown is on the left and the compare button on the right, since that's the last thing the user does.

Reply:
If more dropdowns end up being added to this page in the future though, I'd like to see them organized in the selector card so there's consistency with the layout.

Reply:
> That's an interesting idea. It still seems a bit strange to me to have projects in the card and frameworks as part of the compare button but maybe this is the right way to go since the framework applies to both results/API calls. I suppose you can try it but perhaps switch it so the framework dropdown is on the left and the compare button on the right, since that's the last thing the user does.

Got it! Thanks!